feature-libs/order/components/order-details/order-overview/order-overview-component.service.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { DeliveryMode } from '@spartacus/cart/base/root';
+
+@Injectable()
+export class OrderOverviewComponentService {
+  showDeliveryMode(mode: DeliveryMode | undefined): boolean {
```

Comment:
[MEDIUM]
Let's use `shouldShowDeliveryMode` instead `showDeliveryMode`.
Otherwise, the method name suggest making a side-effectful action (showing something). But in reality it's a boolean indicator _whether_ to show or not.

PS. An analogical comment applies to more places in this PR. Please fix all of them.

Reply:
done!